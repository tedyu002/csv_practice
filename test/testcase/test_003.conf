CSV_INPUT=test_003.conf.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
HEADERS=H1 CHAR(4),H2 VARCHAR(128),H3 VARCHAR(12),H4 INTEGER,H5 BOOL,H6 DOUBLE
CSV_FORMULA=Print H1[5] + H2[3] * 2
SORT_HEADERS=H1,H3
SORT_ORDER=ASC
