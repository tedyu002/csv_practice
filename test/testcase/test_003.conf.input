1997,"Ford ""Escape""",E350,800000,TRUE,12.35
1991,Mercedes-Benz,S320,2800000,TRUE,12.66
2003,Mercedes-Benz,C220,1800000,TRUE,528.69
2016,Volvo,S60,1750000,FALSE,52.63
1991,"Audi Is Not ""Odie"" HaHa!",A8,1850000,TRUE,12.5
2003,Mazda,CX-5,1150000,FALSE,45.96
2017,"Skoda, please add double quote if comma exists",Yeti,950000,FALSE,12.52
2017,Skoda,Rapid,958000,TRUE,15.63
2016,Skoda,Citigo,858000,FALSE,172.8
1991,Audi,Q7,2250000,TRUE,15.69
2016,Volvo,XC90,2550000,TRUE,28.36
2016,Volvo,V40,1550000,FALSE,69.63
2015,Volvo,S80,2350000,FALSE,35.36
2016,"Volvo,,,,, I don't like this",XC60,2350000,FALSE,12.63
2016,Volvo,XC70,2550000,TRUE,29.52
2017,BMW,X1,1550000,TRUE,27.85
2015,BMW,X2,1650000,TRUE,26.36
2017,"BMW """" continue, This is good",X3,1850000,TRUE,36.96
2015,BMW,X4,2050000,TRUE,63.52
2017,BMW,X5,2450000,FALSE,13.52
2017,BMW,X6,2550000,TRUE,23.63
2017,BMW,320i,2250000,FALSE,63.96
2015,"Ferrari, yes, !@#$%^&*(), OOXX",812,4350000,TRUE,125.63
