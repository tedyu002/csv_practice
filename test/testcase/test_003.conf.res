"2003Ford ""Escape""Ford ""Escape"""
