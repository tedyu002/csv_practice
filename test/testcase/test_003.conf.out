"1991","A8","Audi Is Not ""Odie"" HaHa!",1850000,TRUE,12.50
"1991","Q7","Audi",2250000,TRUE,15.69
"1991","S320","Mercedes-Benz",2800000,TRUE,12.66
"1997","E350","Ford ""Escape""",800000,TRUE,12.35
"2003","C220","Mercedes-Benz",1800000,TRUE,528.69
"2003","CX-5","Mazda",1150000,FALSE,45.96
"2015","812","Ferrari, yes, !@#$%^&*(), OOXX",4350000,TRUE,125.63
"2015","S80","Volvo",2350000,FALSE,35.36
"2015","X2","BMW",1650000,TRUE,26.36
"2015","X4","BMW",2050000,TRUE,63.52
"2016","Citigo","Skoda",858000,FALSE,172.80
"2016","S60","Volvo",1750000,FALSE,52.63
"2016","V40","Volvo",1550000,FALSE,69.63
"2016","XC60","Volvo,,,,, I don't like this",2350000,FALSE,12.63
"2016","XC70","Volvo",2550000,TRUE,29.52
"2016","XC90","Volvo",2550000,TRUE,28.36
"2017","320i","BMW",2250000,FALSE,63.96
"2017","Rapid","Skoda",958000,TRUE,15.63
"2017","X1","BMW",1550000,TRUE,27.85
"2017","X3","BMW """" continue, This is good",1850000,TRUE,36.96
"2017","X5","BMW",2450000,FALSE,13.52
"2017","X6","BMW",2550000,TRUE,23.63
"2017","Yeti","Skoda, please add double quote if comma exists",950000,FALSE,12.52
